### DISCUSSION TITLE
Arm64 teamTalk server for raspberry pi

### DISCUSSION THREAD

**Discussion by zstanecic** [BearWare/TeamTalk5#2445] [Ideas]
### Description

There was an image of tt5 server for raspbian in version 5.13. Can it be updated to support new raspbian or ubuntu and arm64?ubu

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
You can find client and server for Raspberry Pi OS 12 arm64 here:

- https://bearware.dk/beta/teamtalk-v5.18.0.5153-raspios12-arm64.tgz
- https://bearware.dk/beta/teamtalkpro-v5.18.0.5153-raspios12-arm64.tgz

**Comment by patmu** [comment]
According download page 5.18 is released now.

But I do not find the 5.18 release version for Pi OS 12 arm64 on the download page. Is this release delayed somehow?

**Reply by bear101** [reply]
I've updated the downloads page with Rasp Pi 12.

**Reply by patmu** [reply]
Thank you!
